craft_application/git/__init__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2024 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Git repository utilities."""
+
+from .errors import GitError
+from .models import GitType
+from .git_repo import GitRepo, get_git_repo_type, is_repo
```

Comment:
Since we're moving to make our public interface more obvious, would you mind making each of these files private?

```suggestion
from ._errors import GitError
from ._models import GitType
from ._git_repo import GitRepo, get_git_repo_type, is_repo
```

Reply:
Done in [1e6ac34](https://github.com/canonical/craft-application/pull/388/commits/1e6ac34174724905f47d3f7bc0e150375951ece7)

Reply:
Thanks!